### DISCUSSION TITLE
Dynamic Routes - 404 error in production mode

### DISCUSSION THREAD

**Discussion by huzefahusain** [vercel/next.js#11202] [Help]
We have created dynamic routes using express in custom server.js file. our url have dynamic parameters. For example below is the url.

**https://www.mydomain.com/en-kw/men/fashion-tech/2751/c

en-kw ( language and country )
men ( gender)**

But when we build and run on production, I am getting lot of 404 errors in console.

**GET https://www.mydomain.com/_next/static/1584607820331/pages/en-kw/men.js net::ERR_ABORTED 404**

Please let me know how we can fix this as this is highly customized ?

**Comment by ericwallen** [comment]
Having the same problem. Any ideas on how to fix it?

**Comment by tonxxd** [comment]
You have to use `as="/path/to"` prop on Link component when linking to dynamic routes
[https://nextjs.org/docs/api-reference/next/link#dynamic-routes](https://nextjs.org/docs/api-reference/next/link#dynamic-routes 
)
I had the same issue and this solved, hope it helps

**Reply by ericwallen** [reply]
Thank you for the direction. This works as described!

**Comment by sabiqthottoly** [comment]
I am having the same issue
